bin/build_homework_function.sh

```diff
@@ -176,6 +175,9 @@ function build_homework {
     # quit (don't continue on to build other homeworks) if there is a compile error
     make -j 8  > $hw_build_path/log_make_output.txt 2>&1
 
+    # run the schema validator on the complete_config generated by the build process
+    $SUBMITTY_INSTALL_DIR/bin/config_syntax_check.py $hw_source $assignment $semester $course
```

Comment:
On resolving the merge conflict, this line should (like the `make_generated_output.py` one) be placed underneath the if block that checks the make exit code?